package.json

```diff
@@ -4,27 +4,41 @@
   "version": "1.0.0",
   "repository": "https://github.com/source-academy/modules.git",
   "license": "Apache-2.0",
+  "scripts-info": {
+    "module": "Add a new bundle or tab",
+    "lint": "Check code for errors",
+    "lint:fix": "Automatically fix lint errors",
+    "build": "Transpile modules' files from src folder to build folder",
+    "serve": "Start the http-server to serve .js files in build folder",
```

Comment:
Not just `.js` files, its all files from build folder (and also the same directory structure).

Reply:
Good catch, will fix that